types/datum_test.go

```diff
@@ -225,12 +225,17 @@ func TestConvertToFloat(t *testing.T) {
 	}
 }
 
-// mustParseTimeIntoDatum is similar to ParseTime but panic if any error occurs.
-func mustParseTimeIntoDatum(s string, tp byte, fsp int) (d Datum) {
+func mustParseTime(s string, tp byte, fsp int) Time {
 	t, err := ParseTime(&stmtctx.StatementContext{TimeZone: time.UTC}, s, tp, fsp)
 	if err != nil {
 		panic("ParseTime fail")
 	}
+	return t
+}
+
+// mustParseTimeIntoDatum is similar to ParseTime but panic if any error occurs.
```

Comment:
seems this comment can be remove

Reply:
Why? This comment is for the next function, and that function is still used by the test :thinking: .

Reply:
Get it